src/types.ml

```diff
@@ -237,8 +246,8 @@ let rec print_sep = function
 let mscgen (pr:problem): unit =
   let last l = List.nth l (List.length l - 1) in
   let rec mscglobal = function
-  | Send(p, q, x, t, g) -> p ^ "->" ^ q ^ " [label=\"" ^ x ^ " = " ^ show_term t ^ "\"];\n" ^ mscglobal g
-  | Branch(p, q, t, branches) ->
+  | Send(p, q, _, x, t, g) -> p ^ "->" ^ q ^ " [label=\"" ^ x ^ " = " ^ show_term t ^ "\"];\n" ^ mscglobal g
```

Comment:
Why do we just go with the public case in here? - `"->"`

Reply:
Because I don't know if I will break the mscgen if I change it to something else ;)